storage/memory/memory.go

```diff
@@ -48,7 +48,7 @@ func match(key *openfgapb.TupleKey, target *openfgapb.TupleKey) bool {
 
 func (s *staticIterator) Next() (*openfgapb.Tuple, error) {
```

Comment:
I think we can get rid of this iterator in favor of the `storage.staticIterator[*openfgapb.Tuple]` instead. More specifically, `storage.NewStaticTupleIterator(tuples)` is effectively the same thing. I don't see us use the `continuationToken` in the iterator anyways. Can someone confirm for me?

Reply:
>  I don't see us use the continuationToken in the iterator anyways.

The token is used _before_ creating the iterator. So yes, effectively you could replace it with the `NewStaticTupleIterator`. But let's do that in a subsequent PR.